deploy/single_server/templates/deployment.yaml

```diff
@@ -66,16 +68,19 @@ spec:
             - containerPort: 8002
               name: metrics
           livenessProbe:
+            failureThreshold: 30
```

Comment:
Adding failureThreshold, initialDelaySeconds and periodSeconds is good but add {{ .Values.image.livenessFailureThreshold }}, etc. variables to values.yaml and leave the defaults at 5. For initialDelaySeconds use default of 0 (to match k8s default).